bin/merge-open

```diff
@@ -25,7 +25,7 @@ def main(
 
     with xopen(input_genbank_metadata, "r") as fin:
         genbank = pd.read_csv(
-            fin, index_col="strain", low_memory=False, sep="\t"
+            fin, index_col="strain", low_memory=True, sep="\t"
```

Comment:
This still reads the entire file into the DataFrame. We would have to use `chunksize` to process the data in chunks. 

Another way we can do this is to only load the `strain` and `biosample_accession` columns to de-duplicate the RKI data then append the deduplicated RKI data to the original GenBank data outside of this script.

Reply:
Sure, that's fine with me.